basic_network.go

```diff
@@ -782,6 +805,51 @@ func handleTranscodeResponse(nw *BasicVideoNetwork, remotePID peer.ID, tr Transc
 	return ErrHandleMsg
 }
 
+func handleTranscodeSub(nw *BasicVideoNetwork, conn net.Conn, ts TranscodeSubMsg) error {
+	glog.Infof("%v In handleTranscodeSub; received from %v", nw.NetworkNode.ID(), conn.RemotePeer())
+	ok := nw.NetworkNode.VerifyTranscoderSig(ts.BytesForSigning(), ts.Sig, ts.StrmID)
+	if !ok {
```

Comment:
Anything stopping us from returning the error here?

Reply:
Fixed [2] in https://github.com/livepeer/go-livepeer-basicnet/pull/32/commits/bdb52f96dab751294da9bce3fd2a5ba0751dcbfc